unit/java_bytecode/java_bytecode_convert_class/convert_abstract_class.cpp

```diff
@@ -125,14 +88,10 @@ SCENARIO("java_bytecode_convert_abstract_class",
     }
     WHEN("Passing a concrete class that extends an abstract class")
     {
-      java_lang->parse(java_code_stream, "Extendor.class");
-
-      symbol_tablet new_symbol_table;
-      java_lang->typecheck(new_symbol_table, "");
-
-      java_lang->final(new_symbol_table);
-
-      REQUIRE(new_symbol_table.has_symbol("java::Extendor"));
+      const symbol_tablet &new_symbol_table=
+        load_java_class(
+          "Extendor",
```

Comment:
There's a typo in this class name, but the class was added in a previous commit. Consider fixing this, but it's not too important (arguably out-of-scope and does not affect program behaviour)

Reply:
Updated to use `Extender` which is I think what you meant?